mizar/dp/mizar/workflows/builtins/pods/create.py

```diff
@@ -41,44 +42,42 @@ def requires(self):
 
     def run(self):
         logger.info("Run {task}".format(task=self.__class__.__name__))
-        # name in self.param.body['metadata']['name']
-        # namespace in self.param.body['metadata']['namespace']
-        # label in self.param.body['metadata']['label']
-        # annotations in self.param.body['metadata']['annotations']
-
-        # logger.info("name: {}".format(self.param.body['metadata']['name']))
-        # logger.info("namespace: {}".format(
-        #     self.param.body['metadata']['namespace']))
-        # logger.info("label: {}".format(self.param.body['metadata']['labels']))
-        # logger.info("annotations: {}".format(
-        #     self.param.body['metadata']['annotations']))
-        # logger.info("hostNetwork: {}".format(
-        #     self.param.spec.get('hostNetwork', 'False')))
-        # logger.info("hostIP: {}".format(self.param.body['status']['hostIP']))
-        # logger.info("phase: {}".format(self.param.body['status']['phase']))
-        # logger.info("podIPs: {}".format(self.param.body['status']['podIPs']))
 
         spec = {
             'hostIP': self.param.body['status']['hostIP'],
             'name': self.param.body['metadata']['name'],
-            'namespace': self.param.body['metadata']['namespace'],
-            'tenant': '',
-            # TODO (Cathy) in case of arktos
-            # get VPC and net information from annotation
-            'vpc': OBJ_DEFAULTS.default_ep_vpc,
+            'type': 'k8s',
```

Comment:
type refers to provider type k8s or Arktos. Right? Would you please change it to a more specific key (e.g. compute_provider)?